README.md

```diff
@@ -1,66 +1,36 @@
-# Deis Workflow v2
+# Deis Workflow Documentation
 
-[![Build Status](https://travis-ci.org/deis/workflow.svg?branch=master)](https://travis-ci.org/deis/workflow) [![Go Report Card](http://goreportcard.com/badge/deis/workflow)](http://goreportcard.com/report/deis/workflow)
+## About
 
 Deis (pronounced DAY-iss) is an open source PaaS that makes it easy to deploy and manage
 applications on your own servers. Deis builds on [Kubernetes](http://kubernetes.io/) to provide
 a lightweight, [Heroku-inspired](http://heroku.com) workflow.
 
-## Work in Progress
+This repository represents the documentation for Deis Workflow which is the
+second major release of the Platform.
 
-![Deis Graphic](https://s3-us-west-2.amazonaws.com/get-deis/deis-graphic-small.png)
+## Requirements
 
-Deis Workflow v2 is currently in alpha. Your feedback and participation are more than welcome, but be
-aware that this project is considered a work in progress.
+The documentation site requires either a local installation of [mkdocs][] or
```

Comment:
This link to MkDocs isn't connected to a URL, but should point to http://www.mkdocs.org/. Also they seem to prefer `MkDocs` with the StudlyCaps / CamelCase.